Makefile

```diff
@@ -706,8 +706,8 @@ localmachine:
 localsystem:
 	# Wipe existing config, database, and cache: start with clean slate.
 	$(RM) -rf ${HOME}/.local/share/containers ${HOME}/.config/containers
-	PODMAN=$(CURDIR)/bin/podman QUADLET=$(CURDIR)/bin/quadlet bats -T --filter-tags '!ci:parallel' test/system/
-	PODMAN=$(CURDIR)/bin/podman QUADLET=$(CURDIR)/bin/quadlet bats -T --filter-tags ci:parallel -j $$(nproc) test/system/
+	PODMAN=$(CURDIR)/bin/podman QUADLET=$(CURDIR)/bin/quadlet bats -T --filter-tags '!ci:parallel' test/system/ ; out=$?; echo "===LOCALSYSTEM SERIAL EXIT STATUS = $$out"; exit $$out
+	PODMAN=$(CURDIR)/bin/podman QUADLET=$(CURDIR)/bin/quadlet bats -T --filter-tags ci:parallel -j $$(nproc) test/system/ ; out=$?; echo "===LOCALSYSTEM PARALLEL EXIT STATUS = $$out"; exit $$out
```

Comment:
Please see #24698. I am so sorry.

Reply:
Oh, I was not even looking at the right Make rule.. Thank you!